### ISSUE TITLE
Can the length of the graphic block be customized? Example 0001 0002 0003 apply to genesis

### ISSUE THREAD

**Issue by ghost** [Rangi42/tilemap-studio#78]
Can the length of the graphic block be customized?
Example 0001 0002 0003
apply to genesis
![QQ截图20230308212551](https://user-images.githubusercontent.com/50875340/223725138-da15ba8c-5024-4ce7-ae85-52b2af187dee.png)

**Comment by Rangi42** [comment]
Yes, different formats may use more or less than 8 bits to define tile IDs. You can see the supported ones in the Help.

If the Genesis format doesn't match any of those, please describe exactly what the format is and I may add support for it.
